v4/src/redux/actions.js

```diff
@@ -0,0 +1,22 @@
+export const addNode = (parentId, props) => ({
+  type: 'ADD_NODE',
```

Comment:
nit: add these to TYPES

Reply:
Do you mean something like:
```javascript
export const ADD_NODE = 'ADD_NODE';
```

Reply:
Something like this: https://github.com/rustielin/iok/blob/b7a55fa18fcdfd9ab16b4ffb5fa6212a0f01a053/v4/src/types.js#L4 for each of the redux action types